actors/common/src/main/java/org/sunbird/learner/actors/LearnerStateActor.java

```diff
@@ -79,6 +71,117 @@ public void onReceive(Request request) throws Exception {
     }
   }
 
+  public void getCourse(Request request) {
+    String userId = (String) request.getRequest().get(JsonKey.USER_ID);
+
+    Map<String, Object> filter = new HashMap<>();
```

Comment:
ElasticSearch block can be moved to -> UserCoursesService - refer example
UserService for naming. & fucntion defn purpose